CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Here is a template for new release sections
 - clicking on village button will randomly select a cluster in a random state (#175)
 - clicking on state button will randomly select a state
 - ability to download the information about settlements clusters in a csv file (#156)
+- add next and previous buttons (#194)
```

Comment:
```suggestion
- add next and previous buttons (#205)
```

In changelog we refer to PR numbers (you can say it fixed Issue #194)

Reply:
✓